Makefile

```diff
@@ -184,8 +178,6 @@ clean:
 	@make -C test-app clean
 
 distclean: clean
-	@make -C tools/ed25519 clean
```

Comment:
On Max OSX had to change `xargs -x rm -f` to `xargs rm -f`

Reply:
ack